### DISCUSSION TITLE
[Transporter Error Output]: ERROR ITMS-90383: Upload limit reached. The upload limit for your account has been reached. Please wait 1 day and try again.

### DISCUSSION THREAD

**Discussion by nloko** [fastlane/fastlane#19949] [Q&A]
Hi all! Running into this error when pushing apps to the store: 

`[Transporter Error Output]: ERROR ITMS-90383: Upload limit reached. The upload limit for your account has been reached. Please wait 1 day and try again.`

Any workarounds available here other than:
- creating another account under new legal entity
- pushing less apps

Also, does anyone know what the official limit is? I see 150 per day documented here: https://docs.fastlane.tools/actions/deliver/

**Comment by joshdholtz** [comment]
I've totally run into this myself when I was fixing `deliver` after it broke one of the WWDCs 😱  I _think_ this limited by user so I _think_ creating another user got me around this? But I don't remember 😅

**Comment by adityapratap12** [comment]
One solution could be to distribute your apps to different app stores. I am currently moving my apps to different appstore and balance it out
